tests/unit/amo/components/TestRatingManager.js

```diff
@@ -264,6 +290,16 @@ describe(__filename, () => {
     expect(props.review).toEqual(userReview);
   });
 
+  it('sets an undefined UserRating review when none exists', () => {
+    const UserRatingStub = sinon.spy(() => <div />);
+
+    render({ UserRating: UserRatingStub, userReview: null });
+
+    sinon.assert.calledWithMatch(UserRatingStub, {
+      review: undefined,
+    });
+  });
```

Comment:
ah, here is the test case related to the `|| undefined` above I guess. I am not sure why `undefined` is better than `null` though: is it only for respecting the `Props` definition of the `UserRating` component? We could also accept `null` as a valid value.

Reply:
The `null` vs `undefined` pattern isn't really anything new. It's the same thing we do in other components with one key difference: loading a `null` (or empty) `userReview` is OK. In most other components, loading empty data renders a 404. Thus, they don't handle the case of rendering with empty data.